Aspire Budgeting/Views/AddTransactionView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AddTransactionView.swift
+//  Aspire Budgeting
+//
+//  Created by TeraMo Labs on 12/8/19.
+//  Copyright © 2019 TeraMo Labs. All rights reserved.
+//
+
+import SwiftUI
+
+struct AddTransactionView: View {
+  @EnvironmentObject var sheetsManager: GoogleSheetsManager
+  
+  @State private var amountString = ""
+  @State private var date = Date()
+  @State private var category = ""
+  
+    var body: some View {
+      ScrollView {
+        AmountTextField(amount: $amountString)
+      }.background(Colors.aspireGray)
+        .edgesIgnoringSafeArea(.all)
+        .onTapGesture {
+        UIApplication.shared.sendAction(#selector(UIResponder.resignFirstResponder), to: nil, from: nil, for: nil)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/696307fdd1c5f01cc8092d49fd19d88beb57f8ec/Aspire Budgeting/Views/AddTransactionView.swift#L24"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>AddTransactionView.swift:24</code></td>
    </tr>
  </tbody>
</table>